dev_requirements.txt

```diff
@@ -15,3 +15,4 @@ scipy>=1.6
 thewalrus>=0.19.0
 plotly>=4.5.0
 pylint==2.13.5
+pandas>=2.0

```

Comment:
this was being installed as a core dependency before, but `python setup.py` was failing to do it properly, so I install it ahead of time here instead!